google/cloud/storage/bucket.py

```diff
@@ -464,6 +485,21 @@ def bucket(self):
         """
         return self._bucket
 
+    @property
+    def public_access_prevention(self):
+        """Setting for public access prevention policy. Options are 'unspecified' (default) or 'enforced'.
+            More information can be found at https://cloud.google.com/storage/docs/public-access-prevention
+
```

Comment:
Likewise, include the docs link here.

Reply:
What do you mean by the docs link?

Reply:
@shaffeeullah https://cloud.google.com/storage/docs/public-access-prevention

Reply:
Hmm, you've linked it here -- I must've missed seeing that before.